browser-test/src/question_lifecycle.test.ts

```diff
@@ -188,6 +188,28 @@ describe('normal question lifecycle', () => {
     expect(await adminNames[4].inputValue()).toContain('option4_admin')
   })
 
+  it('shows markdown format correctly in the preview when creating a new question', async () => {
+    const {page, adminQuestions} = ctx
+    const questionName = 'markdown formatted question'
+
+    await loginAsAdmin(page)
+
+    await adminQuestions.createCheckboxQuestion({
+      questionName: questionName,
+      questionText: 'https://google.com **bold**',
+      helpText: '*italic* [link](https://test.com)',
+      options: [
+        {adminName: 'red_admin', text: 'red'},
+        {adminName: 'green_admin', text: 'green'},
+        {adminName: 'orange_admin', text: 'orange'},
+        {adminName: 'blue_admin', text: 'blue'},
+      ],
+    })
+
+    await adminQuestions.gotoQuestionEditPage(questionName)
+    await validateScreenshot(page, 'question-with-markdown-formatted-preview')
```

Comment:
**suggestion (if-minor):** Is there a way to validate some of the content as well as the screenshot? For instance, making sure that the string `"italic"` does not appear in the rendered content? And that `link` is surrounded by an `<a>` tag?

Reply:
Sorry, one of the examples I gave was confusing/incorrect.

What I should have written is: can you make sure that the string `"italic"` is surrounded by the appropriate HTML tags to render it as italics?

Reply:
For sure. I added some validation statements to check the html tags.